modules/op-release-notes-1-0.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assembly:
+//
+// * openshift_pipelines/op-release-notes.adoc
+
+[id="op-release-notes-1-0_{context}"]
+= Release Notes for Red Hat {pipelines-title} Technology Preview 1.0
+
+[id="new-features-1-0_{context}"]
+== New features
+{pipelines-title} Technology Preview (TP) 1.0 is now available on {product-title} 4.4. {pipelines-title} TP 1.0 is updated to support:
+
+* Tekton Pipelines 0.11.3
+* Tekton `tkn` CLI 0.9.0
+* Tekton Triggers 0.4.0
+* ClusterTasks based on Tekton Catalog 0.11
+
+In addition to the fixes and stability improvements, here is a highlight of what’s new in OpenShift Pipelines 1.0.
+
+=== Pipelines new features
+* Support for  v1beta1 API Version.
+* Support for an improved LimitRange. `limitRangeName` is now removed, and TaskRun now accounts for LimitRange on behalf of the user.
+* PipelineResources can now be marked as optional.
+* Support for sharing data between Tasks using Task Results and Task Params.
+* Support to configure Pipelines to not overwrite the `HOME` environment variable and `workingDir` of Steps.
+* Similar to Task Steps, `sidecars` now support script mode.
+* You can now specify a different scheduler name in TaskRun `podTemplate`.
+* Support for Star Array Notation for variable substitution.
+* Tekton Controller can now be configured to monitor an individual namespace.
+* A new description field is now added to the specification of Pipeline, Task, ClusterTask, Resource and Condition.
+* Addition of  proxy parameters to Git PipelineResources.
+
+=== `tkn`CLI new features
+
+* The `describe` sub-command is now added for the following `tkn` resources -- `eventlistener`, `condition`, `triggertemplate`, `clustertask`, and `triggerbinding`.
+* Support added for `v1beta1` to the following commands along with backward comptibility for `v1alpha1` - `clustertask`, `task`, `pipeline`, `pipelinerun`,  and `taskrun`.
+* The following commands can now list from all namespaces using the `--all-namespaces` flag option:
+** `tkn task list`
+** `tkn pipeline list`
+** `tkn taskrun list`
+** `tkn pipelinerun list`
+The output of these commands is also enhanced to display information without headers using `--no-headers` flag option.
```

Comment:
Add a blank line after line 40. 
the `--no-headers` flag option.

Reply:
Please add a + above the blank line so that this sentence aligns to the bullet list above.